src/components/Mission.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MissionTrue from './MissionTrue';
+import MissionFalse from './MissionFalse';
+
+const Mission = (props) => {
+  const {
+    missionId, missionName, missionDescription, missionStatus,
+  } = props;
+  return (
+    <tr>
+      <td>{missionName}</td>
+      <td>{missionDescription}</td>
+
+      {(() => ((missionStatus) ? <MissionTrue id={missionId} /> : <MissionFalse id={missionId} />)
+      )()}
```

Comment:
- [ ] Kindly refactor your conditional rendering using the recommended syntax.

  ![anwar-condit](https://user-images.githubusercontent.com/36542986/162099549-d5fff6f9-8ac4-4860-af7b-8df29b83b4ea.jpg)